### PR TITLE
[ADF-3022] Create automated test for attachment list - Processes

### PR DESCRIPTION
**Pull request by jdosti**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x ] Other... Please describe:
New automation test for Attachment List - Processes

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-3022

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -5,25 +5,14 @@ const path = require('path');
 const {SpecReporter} = require('jasmine-spec-reporter');
```
Comment (eromano) [inline protractor.conf.js]: can you revert the change on this file?

File: e2e/process_attachmentList_actionMenu.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    let jpgFile = new FileModel({
+        location: resources.Files.ADF_DOCUMENTS.JPG.file_location,
+        name: resources.Files.ADF_DOCUMENTS.JPG.file_name
+    });
+    let pdfFile = new FileModel({
+        location: resources.Files.ADF_DOCUMENTS.PDF.file_location,
+        name: resources.Files.ADF_DOCUMENTS.PDF.file_name
+    });
+
+    let downloadedJpgFile = path.join(__dirname, '\\downloads', jpgFile.name);
```
Comment (eromano) [inline e2e/process_attachmentList_actionMenu.e2e.ts]: let downloadedJpgFile = path.join(__dirname, '/downloads', jpgFile.name);
can you change it like that?

Comment (DenisVuyka) [reply e2e/process_attachmentList_actionMenu.e2e.ts]: path.join in node.js takes care about all slashes, it should be
```
path.join(__dirname, 'downloads', jpgFile.name)
```
